disttask/loaddata/dispatcher.go

```diff
@@ -78,14 +85,18 @@ func (h *flowHandle) ProcessNormalFlow(ctx context.Context, handle dispatcher.Ta
 	switch gTask.Step {
 	case Import:
 		h.switchTiKV2NormalMode(ctx, logutil.BgLogger())
-		if err := postProcess(ctx, handle, gTask, logger); err != nil {
+		if err := postProcess(ctx, handle, gTask, taskMeta, logger); err != nil {
 			return nil, err
 		}
 		gTask.State = proto.TaskStateSucceed
 		return nil, nil
 	default:
```

Comment:
can we give the zero value a name? a solitary `Import` is strange 😂 
Also the code is less readable because `postProcess` is put above `preProcess`. Maybe

```
switch ... {
case ...:
  preProcess
  return ...
case Import:
  ...
  postProcess
}
```